examples/parallel_memmap.py

```diff
@@ -23,43 +99,75 @@ def sum_row(input, output, i):
     output[i] = sum_
```

Comment:
Maybe reuse (the idead of) the above function and just adapt it to write to an output array?